test/mocks.ts

```diff
@@ -0,0 +1,25 @@
+///<reference path="testReference.ts" />
+
+module Mocks {
```

Comment:
It doesn't belong there since it's test code, rather than part of the main library.